src/WooCommerce/Plugin.php

```diff
@@ -37,6 +72,6 @@ public function addGateway($array)
 	}
 
 	public function is_valid_for_use() {
-		return in_array( get_woocommerce_currency(), apply_filters( 'raipay_woocommerce_supported_currencies', array( 'USD', 'XRB' ) ) );
+		return in_array( get_woocommerce_currency(), apply_filters( 'raipay_woocommerce_supported_currencies', $supported_currencies ) );
```

Comment:
XRB is now missing

Reply:
Lol, nice catch - I just copy/pasted the table from the raipay docs and munged it in vim, didn't think that XRP wouldn't be in there. Fixed!